libpod/pod_api.go

```diff
@@ -77,7 +77,7 @@ func (p *Pod) Start(ctx context.Context) (map[string]error, error) {
 // containers. The container ID is mapped to the error encountered. The error is
 // set to ErrCtrExists
 // If both error and the map are nil, all containers were stopped without error
-func (p *Pod) Stop(ctx context.Context, cleanup bool) (map[string]error, error) {
+func (p *Pod) Stop(ctx context.Context, cleanup bool, timeout int) (map[string]error, error) {
```

Comment:
Can we keep Stop() with no timeout argument, and have a StopWithTimeout with the argument? basically just hardcode Stop to call StopWithTimeout(-1)?

Reply:
sure, looking for clarification though on what you  mean by "keep"? Basically take what there is now and call it StopWithTimeOut and make a new func Stop that calls it?